### ISSUE TITLE
`Calicoctl container profile (append...)` has unexpected output when no profile is specified

### ISSUE THREAD

**Issue by TrimBiggs** [projectcalico/calico-docker#559]
This command has the exact same functionality as the `calicoctl endpoint <ENDPOINT_ID> profile (append|set|remove) [<PROFILES>...]` command, which should also require the profiles flag as mentioned in #557.

```
# This does not actually add any profiles.
$ calicoctl container workload_A profile append
Profiles  appended

$ calicoctl endpoint d79123c4784511e5bd1a080027f532f6 profile show
+------+
| Name |
+------+
| PROF |
+------+

# This doesn't remove any profiles, but still succeeds.
$ calicoctl container workload_A profile remove
Profiles  removed.

$ calicoctl endpoint d79123c4784511e5bd1a080027f532f6 profile show
+------+
| Name |
+------+
| PROF |
+------+

# This sets the profile to be an empty string (no profiles)
$ calicoctl container workload_A profile set
Profiles  set.

$ calicoctl endpoint d79123c4784511e5bd1a080027f532f6 profile show
Endpoint has no profiles associated with it.
```

**Comment by robbrockbank** [comment]
Just update the output text is sufficient I think.

**Comment by TrimBiggs** [comment]
The reason the `<PROFILES>` param is optional is to allow the `set` command to use an empty list of profiles to remove all profiles.

The change should be to update the output of the commands to be more clear what is going on.

**Comment by TrimBiggs** [comment]
Fixed by #568
